drivers/SmartThings/matter-media/src/init.lua

```diff
@@ -28,6 +28,13 @@ local configure_handler = function(self, device)
     capabilities.mediaPlayback.commands.play.NAME,
     capabilities.mediaPlayback.commands.pause.NAME,
     capabilities.mediaPlayback.commands.stop.NAME,
+    capabilities.mediaPlayback.commands.rewind.NAME,
+    capabilities.mediaPlayback.commands.fastForward.NAME,
+  }))
+
+  device:emit_event(capabilities.mediaTrackControl.supportedTrackControlCommands({
+    capabilities.mediaTrackControl.commands.previousTrack.NAME,
```

Comment:
Next and previous track are optional commands so we can't assume they are supported. I think we need to remove `mediaTrackControl` from the default profile and then have the read the `Accepted Command List` attribute and if those commands are supported then switch the profile to one that includes `mediaTrackControl`. 

@cjswedes @ctowns can you check my thinking?

Reply:
Yeah, this is the way. I'll note that cluster library support for `AcceptedCommandList` will not be available until the next Lua library release which is preventing us from releasing such a change (similar to the situation that #370 is in).

Reply:
Just to confirm, the plan is to support these by default despite the cluster commands backing them being optional, correct? I am fine with this, just want to make sure we are on the same page.

Reply:
Hey Carter, I am going to separate this work to optionally support the next/prevTrack commands in a separate PR since it can not go in yet